ibtk/include/ibtk/PatchMathOps.h

```diff
@@ -501,6 +516,20 @@ class PatchMathOps : public SAMRAI::tbox::DescribedClass
                     int m = 0,
                     int n = 0) const;
 
+    /*!
+     * \brief Computes dst_l = alpha grad (coef1_m div src1_m) + beta src2_n.
+     */
+    void vc_dilational(SAMRAI::tbox::Pointer<SAMRAI::pdat::SideData<NDIM, double> > dst,
+                       double alpha,
+                       double beta_in,
```

Comment:
1. Can we keep the usual order for coefficients and variables in the interface?
2. What does the suffix "in" mean here?